<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElecteurInterfaceClass</class>
 <widget class="QDialog" name="ElecteurInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ElecteurInterface</string>
  </property>
  <widget class="QLabel" name="label_prenom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Prénom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nas">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TitrePageElecteur">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajout d'un Electeur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bouton_ValiderElecteur">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider le nouvel électeur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_province">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>170</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>2037</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>1970</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_rue">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro de la rue</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom_ville">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenom">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_code_postal">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code postal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom_rue">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de la rue</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code_postal">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_date_naissance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_province">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Province</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom_ville">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de la ville</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_numero_rue">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom_rue">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_nas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Bouton_ValiderElecteur</sender>
   <signal>clicked()</signal>
   <receiver>ElecteurInterfaceClass</receiver>
   <slot>validerElecteur()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerElecteur()</slot>
 </slots>
</ui>
